src/controllers/report.js

```diff
@@ -60,3 +60,13 @@ exports.getReportSummary = function ({ user, query }, res, next) {
         .then(getResponseHandler(res))
         .catch(next)
 }
+
+exports.dismissReport = function ({ user, params }, res, next) {
+    if (user.permissions < PERMISSIONS_MOD) {
```

Comment:
this check shouldn't be needed because isModOrHigher middleware is run on the route